src/authHandler.js

```diff
@@ -0,0 +1,60 @@
+const hapiJWT = require('hapi-auth-jwt2');
+
+const generateAuth = require('./authGenerator').authGenerator;
+
+let sequelize;
+let authConfig;
+// bring your own validation function
+const validate = async (decoded) => {
```

Comment:
Why do we need a validate method when we are already verifying with password?

Reply:
This if for validating the token for each request

Reply:
My bad 🤦‍♂️